generators/contract/templates/javascript/lib/contract-metadata/metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://fabric-shim.github.io/contract-schema.json",
+    "info": {
+        "title": "<%= name %>",
+        "description": "<%= description %>",
+        "termsOfService": "http://example.com/terms/",
```

Comment:
We've got `http://www.example` and no `http://example` in the same `info` block.